core/server/server.spec.js

```diff
@@ -197,22 +202,19 @@ describe('The server', function () {
     })
 
     it('should return the 410 badge for obsolete formats', async function () {
-      const { statusCode, body } = await got(`${baseUrl}npm/v/express.jpg`, {
-        throwHttpErrors: false,
-      })
+      const { statusCode, body } = await got(
+        `${baseUrl}badge/foo-bar-blue.jpg`,
+        {
+          throwHttpErrors: false,
+        }
+      )
       // TODO It would be nice if this were 404 or 410.
       expect(statusCode).to.equal(200)
       expect(body)
         .to.satisfy(isSvg)
         .and.to.include('410')
         .and.to.include('jpg no longer available')
     })
-
-    it('should return cors header for the request', async function () {
-      const { statusCode, headers } = await got(`${baseUrl}npm/v/express.svg`)
-      expect(statusCode).to.equal(200)
-      expect(headers['access-control-allow-origin']).to.equal('*')
-    })
```

Comment:
I am a tad worried that in dropping this test we're potentially losing validation of header behavior. Would you mind elaborating on your thinking behind this?

Reply:
Sorry. I hoped this was clear from the commit message on 202805d but I should have left a comment on the diff.

Before this PR we had 2 copies of the same test in this file:
https://github.com/badges/shields/blob/3efe143edc17a3bd16d07b532ee0021c3485ff35/core/server/server.spec.js#L67-L71
https://github.com/badges/shields/blob/3efe143edc17a3bd16d07b532ee0021c3485ff35/core/server/server.spec.js#L211-L215
I noticed it while I was doing this so I removed the duplicate instead of updating both copies